bin/executor

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+php -e /usr/src/code/app/executor.php -dopcache.preload=opcache.preload=/usr/src/code/app/preload.php
```

Comment:
This file is not required since we moved the executor to open-runtimes.

Reply:
removed